invenio_oauthclient/contrib/cern_openid.py

```diff
@@ -402,3 +409,21 @@ def on_identity_loaded(sender, identity):
         OAUTHCLIENT_CERN_OPENID_SESSION_KEY,
     )
     identity.provides.update(session.get(key, []))
+
+
+class CERNOpenIDLogoutView(LogoutView):
+    """Logout with redirect to logout page of the CERN OPENID provider."""
+
+    def logout_user(self):
+        """Return the logout response of flask-security."""
+        return logout()
+
+    def get(self):
+        """Logout a user."""
+        return self.logout_user()
+
+
+@cern_oauth_blueprint.route('/cern_openid/logout/')
+def logout_redirect():
+    """Redirect to the CERN OpenID logout."""
+    return redirect(current_app.config['OAUTH_REMOTE_REST_APP']['logout_url'])
```

Comment:
What is the code that will return?
```suggestion
    return redirect(current_app.config['OAUTH_REMOTE_REST_APP']['logout_url'], code=302)
```

Reply:
I think is the case already when you return the `redirect` :)

Reply:
@topless Actually, you might be right 😅

Reply:
It is `302` by default... but yeah better safe than sorry :)